app/main.js

```diff
@@ -48,7 +48,6 @@ const persistCurrentColor = exports.persistCurrentColor = (newCurrent) => {
     .then(({ saved }) => {
       let updatedStorage = { current: newCurrent, saved };
       storage.set('saved-colors', updatedStorage)
-        .then(() => console.log('Nailed it!'))
```

Comment:
Electron storage is promised based & looks like you removed some of the .then's here. I'd leave these for now. We could eventually add something like a 'showAlert' function that alerts the user that their color has been saved. Do you agree?